src/software/Localization/main_rigLocalizer.cpp

```diff
@@ -249,24 +253,30 @@ int main(int argc, char** argv)
 #endif
                                            );
     param = new localization::VoctreeLocalizer::Parameters();
```

Comment:
Yes, I forgot all the `delete`...
I will use the `std::unique_ptr.`

@fabiencastan Beginner question: since `std::make_unique` is c++14 shall I do something like
```c++
std::unique_ptr<localization::ILocalizer> localizer;
if(...)
{
   std::unique_ptr<localization::VoctreeLocalizer> tmp (new localization::VoctreeLocalizer(....));
   localizer = std::move(tmp);
}
```

:sweat: verbosity++